src/styles/blocks/header.scss

```diff
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
The 'body' styles should also be in a separate file (e.g., 'base.scss') and not in 'header.scss', as they are not specific to the header block.